docs/workflows/api-token.md

```diff
@@ -0,0 +1,400 @@
+if err != nil {
+  log.Fatal(err)
+}
+resp, err := client.Do(req)
+if err != nil {
+  log.Fatal(err)
+}
+defer resp.Body.Close()
+bodyText, err := ioutil.ReadAll(resp.Body)
+if err != nil {
+  log.Fatal(err)
+}
+fmt.Printf("%s\n", bodyText)
+}
+```
+
+</TabItem>
+</Tabs>
+
+2. Create an access token the with authorization code:
+
+<Tabs groupId="api-token-platform" queryString>
+<TabItem value="curl" label="Curl">
+
+```bash title="/token"
+curl https://auth-$REGION.beyondidentity.com/v1/tenants/$TENANT_ID/realms/$REALM_ID/applications/$MANAGEMENT_APPLICATION_ID/token \
+-X POST \
+-H "Content-Type: application/x-www-form-urlencoded" \
+-d "grant_type=authorization_code&code=$CODE&scope=$SCOPES&client_id=$MANAGEMENT_API_CLIENT_ID&code_verifier=$OPTIONAL_CODE_VERIFIER"
+```
+
+</TabItem>
+<TabItem value="node" label="Node">
+
+```jsx title="/token"
+fetch(
+  'https://auth-REGION.beyondidentity.com/v1/tenants/TENANT_ID/realms/$REALM_ID/applications/MANAGEMENT_APPLICATION_ID/token',
+  {
+    method: 'POST',
+    body: new URLSearchParams({
+      grant_type: 'authorization_code',
+      code: 'CODE',
+      scope: 'SCOPES',
+      client_id: 'MANAGEMENT_API_CLIENT_ID',
+      code_verifier: 'OPTIONAL_CODE_VERIFIER',
+    }),
+  }
+);
+```
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+```python title="/token"
+import requests
+
+data = {
+'grant_type': 'authorization_code',
+'code': 'CODE',
+'scope': 'SCOPES',
+'client_id': 'MANAGEMENT_API_CLIENT_ID',
+'code_verifier': 'OPTIONAL_CODE_VERIFIER',
+}
+
+response = requests.post(
+'https://auth-REGION.beyondidentity.com/v1/tenants/TENANT_ID/realms/$REALM_ID/applications/MANAGEMENT_APPLICATION_ID/token',
+data=data,
+)
+```
+
+</TabItem>
+<TabItem value="go" label="Go">
+
+```go title="/token"
+package main
+
+import (
+"fmt"
+"io/ioutil"
+"log"
+"net/http"
+"strings"
+)
+
+func main() {
+client := &http.Client{}
+var data = strings.NewReader(`grant_type=authorization_code&code=CODE&scope=SCOPES&client_id=MANAGEMENT_API_CLIENT_ID&code_verifier=OPTIONAL_CODE_VERIFIER`)
+req, err := http.NewRequest("POST", "https://auth-REGION.beyondidentity.com/v1/tenants/TENANT_ID/realms/$REALM_ID/applications/MANAGEMENT_APPLICATION_ID/token", data)
+if err != nil {
+  log.Fatal(err)
+}
+req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
+resp, err := client.Do(req)
+if err != nil {
+  log.Fatal(err)
+}
+defer resp.Body.Close()
+bodyText, err := ioutil.ReadAll(resp.Body)
+if err != nil {
+  log.Fatal(err)
+}
+fmt.Printf("%s\n", bodyText)
+}
+```
+
+</TabItem>
+</Tabs>
+
+### Revoking Access Tokens:
+
+In order to revoke an access token, you must be passed authentication in the form of either Bearer or Basic.
+
+In the case of Basic authentication, the passed authentication must come from an Application registered in auth-config-service. The passed access token must be signed by the same client id as the application.
```

Comment:
I'd suggest rephrasing this:

>  In the case of Basic authentication, the passed authentication must come from the confidential Application that the token has been minted for.

I don't think it's useful to say that the application is registered in auth-config-service.

Reply:
I agree, let's update the source specs too to prevent confusion.